apps/cli/src/compose/rollups/proxy/rollups-node.yaml

```diff
@@ -0,0 +1,10 @@
+http:
+    routers:
+        inspect_server:
+            rule: "PathPrefix(`/inspect`)"
+            service: inspect_server
+    services:
+        inspect_server:
+            loadBalancer:
+                servers:
+                    - url: "http://rollups-node:10012/inspect"
```

Comment:
This is why I didn't use file and used inline content in YAML files, so we could replace values with environment variables, like this `10012` for `CARTESI_INSPECT_PORT`, for ex.

Reply:
I understand, but I think a lot of env vars we should stop using, and hard code for simplicity.